src/main/java/com/codejoust/main/dto/game/SubmissionRequest.java

```diff
@@ -13,4 +13,5 @@
     private String code;
     private String input;
     private UserDto initiator;
+    private int problem = 0;
```

Comment:
nit: rename this to index or problemIndex or something (since problem is usually used with a Problem or ProblemDto class

Reply:
be sure to push the change if you haven't already